config.example.storage.wikimedia.yaml

```diff
@@ -0,0 +1,113 @@
+# RESTBase wikimedia example config
+
+# Load some project templates. These are referenced / shared between domains
+# in the root_spec further down.
+default_project: &default_project
+  x-modules:
+    - spec:
+        paths:
+          /{api:v1}:
+            x-modules:
+              - spec:
+                  paths:
+                    /parsoid_bucket:
+                      x-modules:
+                        - path: sys/multi_content_bucket.js
+                          options:
+                            grace_ttl: 1
+                            index_ttl: 864000
+                            renew_expiring: true
+                            delete_probability: 1
+                            table_name_prefix: parsoid_ng
+                            main_content_type:
+                              name: html
+                              value_type: blob
+                            dependent_content_types:
+                              - name: data-parsoid
+                                value_type: json
+                    /key_value:
```

Comment:
I think we'll need here only `key_value` and `page_revisions` once the simplification is completed.

Reply:
Yes. But it's not complete yet, so I needed this to make it work. It will go away.